### DISCUSSION TITLE
deploy is waiting for github-pages deployment approval

### DISCUSSION THREAD

**Discussion by sspen094** [community/community#59597] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Im trying to deploy a webpage and that staus is just "[deploy] is waiting for github-pages deployment approval".  I dont see any way to approve it

**Comment by laserdisk** [comment]
+1, seeing the same issue now

**Comment by commputethis** [comment]
I'm seeing the same issue

**Comment by santacalculus** [comment, marked as answer]
I think there's an ongoing issue https://www.githubstatus.com they're trying to fix it. I've been having the same error as well and have caused quite a bit of carnage in my repo trying to fix it lol. decided to wait it out for now

**Reply by thedoggybrad** [reply]
Hopefully, they were able to fix it soon. I am just in a middle of deploying some projects.

Latest:
We are continuing to monitor the rollout of the fix for failing Actions jobs using the deployment environments feature. We are expecting the rollout to take up to another 1 to 1.5 hours.

**Reply by santacalculus** [reply]
Issue is resolved now. If you're still struggling, cancel the pending workflow and push again. Goodnight :)

**Reply by yamonc** [reply]
thank for good answer. but, it did take a bit of time.... 😂

**Reply by viraj63** [reply]
Ohh Great to hear! that the problem is been resolved

**Reply by sspen094** [reply]
not sure if it was fixed separately or if this is a backwards workaround but I added myself as a reviewer. in settings>pages and approved myself and it worked but could be that they just fixed it overnight. Thanks!

**Reply by queenofcorgis** [reply]
Thanks everyone for the patience, updates, reports and workarounds!

**Comment by zhujiem** [comment]
+1

**Comment by yiyunlei** [comment]
Same issue here :(

**Comment by ApprovalPage** [comment]
Same issue

**Reply by charl-potgieter** [reply]
+1

**Comment by yazn-108** [comment]
نفس المشكلة
same issue

**Comment by 5HT** [comment]
👎

**Comment by mtgrunt** [comment]
+1, I'm seeing the same issue.

**Comment by carlosev95** [comment]
I'm seeing the same issue, I even had to reduce the size of the icons 😅. We've to wait.

**Comment by psebastiani2021** [comment]
I've been encountering this problem in my GitHub pages repository, I thought I was the only one 😰

**Comment by xing-eram** [comment]
Same problem here!

**Reply by FarahMunawar** [reply]
same here

**Comment by marioslazaro** [comment]
I'm having the same issue, any updates?

**Reply by santacalculus** [reply]
According to the status page, it might be another 1.5 to 2 hours before the rollout is done

**Comment by writeonlycode** [comment]
Same issue here.

**Comment by TheShiningBoots** [comment]
Same issue

**Comment by thedoggybrad** [comment]
Latest:
We are continuing to monitor the rollout of the fix for failing Actions jobs using the deployment environments feature. We are expecting the rollout to take up to another 1 to 1.5 hours.

**Comment by niusj03** [comment]
Same issue,

**Comment by hayden-t** [comment]
+1

**Comment by bybeye** [comment]
+1

**Comment by ScarlettChen0111** [comment]
+1 🤷‍♀️

**Comment by ablevargheseme** [comment]
+1 same issue

**Reply by cs7eric** [reply]
+ 1

**Comment by KyanChen** [comment]
The same problem

**Comment by JesseYang0528** [comment]
Same issue...

**Comment by ghost** [comment]
+1 Same issue here

**Comment by dendi875** [comment]
+1 same issue

**Comment by sunho** [comment]
same 🦈

**Comment by dcsndevs** [comment]
+1 Same issue here

**Comment by Elfsong** [comment]
Same issue here...

**Comment by coffeezaroo** [comment]
Same issue here

**Comment by kanazuko** [comment]
Hahahah I thought I was not doing correctly the steps. This is my 1st time trying to deploy a react app and I was really frustrated
Thanks 👍

**Reply by ghost** [reply]
Ye same for me

**Reply by ghost** [reply]
Ye same for me

**Comment by yisuchen** [comment]
+1

**Comment by leepuu** [comment]
+1

**Comment by cyao2q** [comment]
+1

**Comment by ssong-yi** [comment]
+1 Same issue here

**Comment by taufik-my** [comment]
+1 ㅠㅠ

**Comment by a-rustacean** [comment]
I was deploying GitHub pages using GitHub actions and thought that was the issue but when I tried manually I still got the same issue, Hope they fix it soon

**Comment by Yaohong9257** [comment]
I have encountered this issue.

**Comment by heum2** [comment]
Is this issue resolved? I'm still having it.

**Reply by heum2** [reply]
It's work!

**Comment by ThreeMonth03** [comment]
+1

**Comment by Patel-Parth12** [comment]
i have also facing this issue

**Comment by TinySnow** [comment]
First time I think it's my workflow problem, and I modified my action file many times.
Until I see this discussion...
QvQ

**Comment by kaonmir** [comment]
I thought this was my workflow or my runner problem... :( Thanks guys.

**Reply by kaonmir** [reply]
It has been resolved 5 minutes ago!

![image](https://github.com/community/community/assets/40649016/df6b7f17-6751-4b91-a675-b59f37124f36)

**Reply by ruihang-chu** [reply]
Why does it still not work for me? While my friend can succesfully update it. Is there anyone still in the waiting process now ?

**Reply by TinySnow** [reply]
Please cancel the current outdated flow and rerun it, everything will be ok.

**Comment by hfwang132** [comment]
same problem here :) still unresolved

**Comment by guoqingxibei** [comment]
It works when I cancelled the blocked workflow and rerun it.

**Reply by hfwang132** [reply]
This worked. Thanks. The issue seems resolved at Jun 29, 06:43 UTC as shown in https://www.githubstatus.com.

**Comment by leahy268** [comment]
I am still unable to cancel the blocked workflow.  I pressed cancel more than once during the day.  Any ideas?

**Reply by TinySnow** [reply]
Refresh your page or change to another browser? give it a try, many people solved.

**Reply by santacalculus** [reply]
The error email is being sent when something new is pushed to the main branch. Have you done that yet?

**Reply by leahy268** [reply]
yes things have been pushed all day to the main branch.  Dev is deploying perfectly and has been all day.  Test however is not.  Ironically Test never did require approval and still doesn't but is blocked waiting for approval

**Comment by ErikJiang** [comment]
![image](https://github.com/community/community/assets/10629406/9a6d3465-d31f-4168-931e-2103c0f22bdb)

**Comment by sunho** [comment]
Fixed!!! I can finally sleep 🛌

**Comment by viraj63** [comment]
Guys the issue has been resolved...Just cancel the last push....and repush it

**Reply by yamonc** [reply]
you can re-Run jobs to re-deploy you application.
![image](https://github.com/community/community/assets/52881289/41755c51-68b8-4b0b-8981-1380c95bde76)

**Reply by viraj63** [reply]
Off course you can do that too....I had other additional deployments so pushed the code that was need and ran it once more..

**Comment by estahn** [comment]
Still stuck in `is waiting for ... deployment approval`. Tried to cancel many times receiving `You have successfully requested the workflow to be canceled.`.

Is there a massive backlog and it takes longer?

**Comment by mulecalle** [comment]
It work's after canceling and re-triggering the job.

**Reply by adammorrell8** [reply]
thanks, that fixed it for me too

**Comment by leahy268** [comment]
Unfortunately I still can't cancel my running job to re-run it.
It is still waiting for deployment approval to Test.  Which has never required approval.  At this point I've tried just about everything.

**Reply by leahy268** [reply]
So even from the api I can't cancel the job or rerun it.  It appears in the pending deployments but I am not able to approve it from the api either.  If I try to I get the response  "No pending deployment requests to approve or reject".  Really don't know where to go from here.  Currently our pipeline is in a dead state.

**Comment by carlosev95** [comment]
It works! Just canceling currently workflow and deploy again.

**Comment by Bluesboy** [comment]
Just ran into the same issue. Canceling job and re-triggering it, solved the issue, but I can't found rule that was requiring approval.
